xcorim-code-points.cddl

```diff
@@ -1,15 +1,24 @@
-; scoped to protected-signed-xcorim-header map
-corim-meta-key = 8
+; unsigned-xcorim-map
+xcorim.creator = 0
+xcorim.deny-list = 1
 
-deny-id = 99
-validity = 121
+; xcorim-entity-map
+xcorim.entity-name = 0
+xcorim.role = 1
+xcorim.reg-id = 2
 
-denied-rims=120
+; $xcorim-role-type-choice
```

Comment:
Better to have the comment on role type choice as the earlier comments for roles

Reply:
agree, but there was no description available in Table 5.5.2 so I left this blank...

Reply:
but in general I'd expect that the CDDL remains as "pure" as possible and we migrate any textual description in the relevant draft section(s).